src/mappings/escrow.ts

```diff
@@ -49,7 +49,7 @@ export function handleCancelThaw(event: CancelThaw): void {
 }
 
 export function handleDeposit(event: Deposit): void {
-    let transaction = new Transaction(event.transaction.hash.toHexString())
+    let transaction = new Transaction(event.transaction.hash.toHexString() + '-' + event.logIndex.toString())
```

Comment:
Not sure if it makes sense, but could this be moved in a function such as `newTransaction(event)` to make it cleaner and less error-prone?

Reply:
Not sure if its the best approach on that, Ill ask Juan about this what he thinks might be best in this regard.

Reply:
Will add it into a new PR